website/docs/docs/deploy/job-scheduler.md

```diff
@@ -67,32 +91,82 @@ Jobs consume a lot of memory in the following situations:
 Refer to [dbt Cloud architecture](/docs/cloud/about-cloud/architecture) for an architecture diagram and to learn how the data flows.
 
 
-<!-- leaving space for CI job treatment, runtime, warm up pods updates
+### Pod memory 
```

Comment:
Created an empty section here. i think we wanted to document current behaviors/treatments here so people can avoid memory failures

Reply:
thanks! We might add but let's leave this out for now. Job memory is what I wanted to write above (job and pod memory are the same)